db/migrate/20240424210712_create_good_job_execution_error_backtrace.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class CreateGoodJobExecutionErrorBacktrace < ActiveRecord::Migration[7.1]
```

Comment:
Is that intentionally commited? We should onboard this somehow, but maybe in other PR? 🤔

Reply:
:information_source: migration onboarded at https://github.com/rubygems/rubygems.org/pull/4647